### PR TITLE
Drop PhantomJS & use Firefox for tests

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Dropping PhantomJS & migrate to FF like we did in web-console.

@jwforres @jeff-phillips-18

### CODE REVIEW

**Comment by jwforres** [comment]
im good with the change in theory but looks like its failing to start firefox in travis

**Comment by benjaminapetersen** [comment]
I'll look at the `travis.yaml` here in just a sec. might be missing something.

**Comment by benjaminapetersen** [comment]
[TEST] `travis.yml` updated

**Comment by benjaminapetersen** [comment]
Think we good here now.  Needed the `xvfb` install stuff.

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,10 @@ cache:
   directories:
   - node_modules
   - bower_components
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```
Comment (jwforres) [inline .travis.yml]: test-headless should be doing this for you already https://github.com/openshift/origin-web-common/blob/master/hack/test-headless.sh#L34

and if you look at our travis.yml in origin-web-console this wasn't necessary

Comment (benjaminapetersen) [inline .travis.yml]: yeah thought so, but this seemed to make it work (is what i copied from other feeder repos like kve & extension-registry).  Ill dig around a bit more.

**Comment by spadgett** [comment]
Recently Travis updates broke origin-web-console tests using Firefox. We probably need to hold off on this.

**Comment by benjaminapetersen** [comment]
Adding the comment/links here as well, [nightmare](https://github.com/segmentio/nightmare) [nightmare](http://www.nightmarejs.org/) could be worth a shot.  I may tinker with it a bit & see if there is any benefit.

**Pull request closed without merging**